node-repository/src/main/java/com/yahoo/vespa/hosted/provision/restapi/NodePatcher.java

```diff
@@ -274,6 +275,8 @@ private Node applyField(Node node, String name, Inspector value, Inspector root,
                 return nodeWithTrustStore(node, value);
             case "wireguardPubkey":
                 return node.withWireguardPubkey(SlimeUtils.optionalString(value).map(WireguardKey::new).orElse(null));
+            case "wireguardKeyTimestamp":
```

Comment:
Why is this patchable? Take the current timestamp in node-repo when the key is updated instead?

Reply:
It did not make sense to patch this separately. Updated to setting it along with the key.

Reply:
Why should it be patchable at all? Generally we don't take timestamps for client for things that are meant to be 'now'.

Reply:
I misunderstood your comment. Will fix.